Helper/GenerateShopperConversionId.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Adyen\Payment\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Adyen\Payment\Helper\Util\Uuid;
+use Magento\Checkout\Model\Session as CheckoutSession;
+
+class GenerateShopperConversionId extends AbstractHelper
+{
+    const SHOPPER_CONVERSION_ID = 'shopper_conversion_id';
+
+    /**
+     * @var CheckoutSession
+     */
+    private $checkoutSession;
+
+    /**
+     * @param Context $context
+     * @param CheckoutSession $checkoutSession
+     */
+    public function __construct(
+        Context $context,
+        CheckoutSession $checkoutSession,
+    ) {
+        parent::__construct($context);
+        $this->checkoutSession = $checkoutSession;
+    }
+
+    /**
+     * Get or generate a ShopperConversionID
+     *
+     * @return string
+     */
+    public function getShopperConversionId(): string
+    {
+        $shopperConversionId = Uuid::generateV4();
```

Comment:
`generateV4` method can throw `RuntimeException`. However, this exception is not handled here.

This will result in failing the whole `/paymentMethods` request. IMO, this needs to be handled silently without failing the API call and causing negative impact on the downstream operations like `/payments` API call. What would you think about logging an error message and returning `null` if an exception is thrown here?